compiler/il/AliasSetInterface.hpp

```diff
@@ -284,57 +278,93 @@ class TR_SymAliasSetInterface : public TR_AliasSetInterface<TR_SymAliasSetInterf
       {
       TR::Compilation *comp = TR::comp();
       LexicalTimer t("removeAllAliases", comp->phaseTimer());
-      TR_ASSERT(_aliasSetType == useDefAliasSet || _aliasSetType ==  UseOnlyAliasSet, "failed: can only call removeAllAliases for UseOnly or useDef presently");
+      TR_ASSERT(_AliasSetInterface == useDefAliasSet || _AliasSetInterface ==  UseOnlyAliasSet, "failed: can only call removeAllAliases for UseOnly or useDef presently");
```

Comment:
should be TR_ASSERT_FATAL IMO to make sure this is checked...

Reply:
Have changed to `TR_ASSERT_FATAL` in 091c9f0